src/components/core/tooltip/arrow-tooltip/arrow-tooltip.tsx

```diff
@@ -0,0 +1,206 @@
+import { ReactElement } from 'react';
+import classNames from 'classnames';
+import * as Stitches from '@stitches/react';
+import CurvedArrowIcon from '@assets/svg/curved-arrow.svg';
+import StraightArrowIcon from '@assets/svg/straight-arrow.svg';
+import useTheme from '@core/theme-context/use-theme';
+import { styled } from 'Styles/stitches.config';
+import { modifyVariantsForStory } from 'Styles/type-utils';
+
+type ArrowDirection =
+    | 'top_left'
+    | 'top_center'
+    | 'top_right'
+    | 'right_top'
+    | 'right_center'
+    | 'right_bottom'
+    | 'bottom_right'
+    | 'bottom_center'
+    | 'bottom_left'
+    | 'left_top'
+    | 'left_center'
+    | 'left_bottom';
+
+type ArrowTooltipProps = {
+    arrow_direction?: ArrowDirection;
+    arrow_color?: string;
+    arrow_size?: number;
+    class_name?: string;
+    class_name_arrow?: string;
+    icon?: string;
+    is_fixed_width?: boolean;
+    children: ReactElement;
```

Comment:
> ReactElement is the type for elements in React, either created via JSX or React. createElement. ReactNode is wider, it can be text, number, boolean, null, undefined, a portal, a ReactElement, or an array of ReactNodes. It represents anything that React can render.
@vinu-deriv what do you think about using `ReactNode` instead of `ReactElement`?